samples/MusicStore/Models/SampleData.cs

```diff
@@ -21,14 +21,15 @@ public static class SampleData
         {
             using (var serviceScope = serviceProvider.GetRequiredService<IServiceScopeFactory>().CreateScope())
```

Comment:
Since you're here, fix this to use the new extension method `CreateScope` on `IServiceProvider`